head_segmentation/_version.py

```diff
@@ -1,2 +1,3 @@
 # __version__ = "1.0.0"  # First version
-__version__ = "1.1.0"  # Segmentation model downloaded from GDrive
+# __version__ = "1.1.0"  # Segmentation model downloaded from GDrive
+__version__ = "1.3.0"  # add gpu inference
```

Comment:
I'd like to move changelog from the `_version.py` file to release notes. Can I ask you for a favour: would you remove those commented `__version__`'s tags. I'm looking for having a file which looks something like (contains only such single line as it's content):
```suggestion
__version__ = "1.3.0"
```

Reply:
Happy to do something for this good project 🚀. If there are any problems, contact me and I will do it.

Reply:
Thanks @9527-csroad 🙌. Lemme collect my thoughts over a weekend and I'll spawn a bunch of issues that may improve the project.